### PR TITLE
MGMT-13325: MGMT-13300: Assisted-Installer: Add support with 4.13 dev preview images

### PR DESCRIPTION
**Pull request by eliorerz**
Add jobs for OCP 4.13, periodic ones will be added in the future after verifying that it working as expected.

/cc @osherdp @adriengentil

### CODE REVIEW

**Review by adriengentil — APPROVED** [review]

File: ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml [hunk 1]
```diff
@@ -856,6 +856,14 @@ tests:
         OPENSHIFT_VERSION=4.12
         OPENSHIFT_INSTALL_RELEASE_IMAGE=""
     workflow: assisted-baremetal
+- as: e2e-metal-assisted-4-13
```
Comment (osherdp) [inline ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml:859]: maybe also make it optional for now (like in test-infra)

**Review by osherdp — CHANGES_REQUESTED** [review]

File: ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml (hunk 1 above)
Comment (eliorerz) [reply ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml:859]: Updated. Thanks

**Review by osherdp — APPROVED** [review]

**Pull request merged**